knative/deployer.go

```diff
@@ -44,13 +44,13 @@ func (d *Deployer) Deploy(f faas.Function) (err error) {
 		output = &bytes.Buffer{}
 	}
 
-	c, err := root.NewRootCommand(nil)
-	if err != nil {
-		return err
-	}
-	c.SetOut(output)
+	params := commands.KnParams{}
+	params.Initialize()
+	c := service.NewServiceCreateCommand(&params)
+	// Print standard kn service create output aka progress of creation
```

Comment:
I like the consistency with `kn`.

Reply:
That would be nice tho', in other universe maybe... :)